### PR TITLE
LPD-20530 Create the service context taking into account the object's entry status

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-20530

### CODE REVIEW

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java [hunk 1]
```diff
@@ -1240,6 +1243,44 @@ null, _getEndpoint(objectRelationship2.getName()),
 		_assertEquals(_objectEntry2, jsonArray);
 	}
 
+	@Test
+	public void testPutObjectEntryRelatedObjectEntryWithDraftStatus()
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java]: @marcelabc I updated the code 😃

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
The [testSortByOneToManyRelationshipSystemObjectFields](https://testray.liferay.com/#/project/34754/routines/34756/build/48030476/case-result/48032860) failure is related

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
> The [testSortByOneToManyRelationshipSystemObjectFields](https://testray.liferay.com/#/project/34754/routines/34756/build/48030476/case-result/48032860) failure is related

Fixed!

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**